lib/resource/InstanceResource.js

```diff
@@ -55,4 +57,51 @@ InstanceResource.prototype.delete = function deleteResource(callback) {
   this.dataStore.deleteResource(this, callback);
 };
 
+InstanceResource.prototype.invalidateCache = function invalidateResource(callback) {
```

Comment:
I prefer that the property name has the same name as the function.  Also, could we simply call this `invalidate`?